packages/picasso.js/src/core/chart-components/axis/axis-label-size.js

```diff
@@ -59,7 +59,7 @@ function shouldAutoTilt({
 }) {
   const glyphCount = settings.labels.maxGlyphCount;
   const m = state.labels.activeMode === 'layered' ? 2 : 1;
-  const magicSizeRatioMultipler = 0.7; // So that if less the 70% of labels are visible, toggle on tilt
+  const magicSizeRatioMultipler = settings.labels.tiltThreshold ? settings.labels.tiltThreshold : 0.7; // So that if less the 70% of labels are visible, toggle on tilt or use variable tiltThreshold
```

Comment:
Add `tiltThreshold ` as a default axis setting in `axis-default-settings.js`

Reply:
@nguyenlam1 don't forget to run `build` to update `docs/scriptappy.json`

Reply:
@miralemd good catch! :)